SPECS/sm.spec

```diff
@@ -312,6 +312,12 @@ systemctl start sr_health_check.timer
 # However it won't start without linstor-controller.service
 systemctl enable linstor-monitor.service
 
+# XCP-ng: We must reload the multipathd configuration without restarting the service to prevent
+# the opening of /dev/drbdXXXX volumes. Otherwise if multipathd opens a DRBD volume,
+# it blocks its access to other hosts.
+# This command if also important if our multipath conf is modified for other drivers.
+multipathd reconfigure
```

Comment:
If this can work when multipathd is not running, then all that is required is a `Requires(post):` dependency to ensure the package that contains multipathd is present.

Otherwise, we must test first whether it is running. Executing this only when updating the package rather than at initial installation could work, but if someday we start updating packages right after the initial installation, then this will cause issues. So testing whether multipathd is running, if can be done simply, might be better.